src/zc/buildout/tests.py

```diff
@@ -35,6 +38,116 @@
     os_path_sep *= 2
 
 
+class TestEasyInstall(unittest.TestCase):
+
+    # The contents of a zipped egg, created by setuptools:
+    # from setuptools import setup
+    # setup(
+    #    name='TheProject',
+    #    version='3.3',
+    # )
+    #
+    # (we can't run setuptools at runtime, it may not be installed)
+    EGG_DATA = (
```

Comment:
I would just keep the zipped file in source control directly instead of embedding a string representation of the zip file in source code.

Another alternative is to keep the zipped egg unzipped as a subdirectory, and use the zipfile module to recompose the zipped egg.

Reply:
Ping 😄 

An issue I thought of with keeping the zipped egg in source control is that there is currently no tests package, just a module. So either it just goes in among the source code, cluttering it, or the module gets refactored to a package, complicating the diff.